testing/integration/otel_helm_test.go

```diff
@@ -55,22 +54,36 @@ func TestOtelKubeStackHelm(t *testing.T) {
 	}
 
 	testCases := []struct {
-		name                       string
-		helmReleaseName            string
-		valuesFile                 string
-		atLeastValidatedPodsNumber int
+		name  string
+		steps []k8sTestStep
 	}{
 		{
-			name:            "helm standalone agent default kubernetes privileged",
-			helmReleaseName: "kube-stack-otel",
-			valuesFile:      "../../deploy/helm/edot-collector/kube-stack/values.yaml",
-			// - A Daemonset to collect K8s node's metrics and logs
-			// (1 EDOT collector pod per node)
-			// - A Cluster wide Deployment to collect K8s metrics and
-			// events (1 EDOT collector pod per cluster)
-			// - An OpenTelemetry Operator Deployment (1 pod per
-			// cluster)
-			atLeastValidatedPodsNumber: 3,
+			name: "helm kube-stack operator standalone agent kubernetes privileged",
+			steps: []k8sTestStep{
+				k8sStepCreateNamespace(),
+				k8sStepHelmDeployWithValues(chartLocation, "kube-stack-otel",
```

Comment:
```suggestion
				k8sStepHelmDeployWithValueOptions(chartLocation, "kube-stack-otel",
```

Reply:
renamed in https://github.com/elastic/elastic-agent/pull/6444/commits/ab64abd4ce2600177c9be5f230f9eca2ce71f8b8